app/src/main/java/com/android/unio/ui/event/overlay/PictureOverlay.kt

```diff
@@ -99,13 +201,80 @@ fun PictureOverlay(
                           .testTag(EventDetailsTestTags.EVENT_PICTURES_ARROW_RIGHT),
                   colors =
                       IconButtonDefaults.iconButtonColors(
-                          contentColor = MaterialTheme.colorScheme.onPrimary,
-                          containerColor = MaterialTheme.colorScheme.primary)) {
+                          contentColor = MaterialTheme.colorScheme.onPrimary)) {
                     Icon(
                         Icons.AutoMirrored.Filled.ArrowForward,
                         context.getString(R.string.event_details_content_description_arrow_right),
-                        modifier = Modifier.size(iconSize))
+                        modifier = Modifier.size(arrowSize))
+                  }
+
+              Row(
+                  modifier =
+                      Modifier.testTag(EventDetailsTestTags.INTERACTION_ROW)
+                          .align(Alignment.BottomCenter)
+                          .fillMaxWidth()
+                          .padding(horizontal = PADDING_VALUE),
+                  horizontalArrangement = Arrangement.SpaceBetween,
+                  verticalAlignment = Alignment.CenterVertically) {
+                    Row(verticalAlignment = Alignment.CenterVertically) {
+                      IconButton(
+                          onClick = onClickLike,
+                          modifier =
+                              Modifier.testTag(EventDetailsTestTags.EVENT_PICTURES_LIKE_BUTTON),
+                          colors =
+                              IconButtonDefaults.iconButtonColors(
+                                  contentColor = MaterialTheme.colorScheme.onPrimary)) {
```

Comment:
Consider hardcoding the colors because as you said, it seems better when being in light mode